xbmc/cores/AudioEngine/Interfaces/AE.h

```diff
@@ -255,5 +255,7 @@ class IAE
    * Indicates if dsp addon system is active.
    */
   virtual bool HasDSP() { return false; };
+
+  virtual bool GetCurrentSinkFormat(AEAudioFormat &SinkFormat) { return false; }
```

Comment:
Never implement anything that is currently not needed. 
1)
There is no other audio engine
2)
the method does only return true atm

If 1) or 2) change, then we can recondider